builder/README.md

```diff
@@ -14,3 +14,12 @@ $ npm install
 ```bash
 $ npm start
 ```
+
+#### Docker
+
+Bind a docker daemon to the builder so he can run nbuild container,
+it also needs two volumes, for manifests and packages and needs to be a tty
+
+```sh
+docker run -v /var/run/docker.sock:/var/run/docker.sock -v nbuild_manifests:/home/node/builder/manifests/ -v nbuild_out:/home/node/builder/out/ --net=host -t tmp/builder
```

Comment:
Why do you need --net=host? It actually exposes the host network to the container, which is kinda meh :/

(also, edit the path to use /app instead of /home)